File: php/examples/getCovers.php
<?php

// Serie can be the ID, the Name or if the series is already loaded the array
$serie = 'Dragonball';

require_once '../src/BurningSeries.php';
$bs = new BurningSeries();

$cover = $bs->getCover($serie);

echo "<img src=\"{$cover}\" alt=\"Cover for {$serie}\" />";

File: java/examples/de/jonesboard/burningseries/examples/getCovers.java
package de.jonesboard.burningseries.examples;

import de.jonesboard.burningseries.BurningSeries;

public class getCovers {

	public static void main(String[] args) {
		// Serie can be the ID, the Name or if the series is already loaded the array
		String serie = "Dragonball";

		BurningSeries bs = new BurningSeries();

		try {
			String coverUrl = bs.getCover(serie);
	
			System.out.println("Cover for " + serie + ";\tLink: " + coverUrl);
		} catch(Exception e) {
			e.printStackTrace();
		}
	}


}
